execution/src/rpc/http_rpc.rs

```diff
@@ -79,91 +70,60 @@ impl ExecutionRpc for HttpRpc {
             .as_ref()
             .map(|data| Bytes::from(data.as_slice().to_owned()));
 
-        let tx = TypedTransaction::Eip1559(raw_tx);
-        let list = self
-            .provider
-            .create_access_list(&tx, block)
-            .await
-            .map_err(|e| RpcError::new("create_access_list", e))?;
+        let block = serde_json::to_value(BlockId::from(block))?;
+        let tx = serde_json::to_value(TypedTransaction::Eip1559(raw_tx))?;
 
-        Ok(list.access_list)
-        */
-        todo!()
+        Ok(self
+            .request("eth_createAccessList", [tx, block])
+            .await
+            .map_err(|e| RpcError::new("create_access_list", e))?)
     }
 
     async fn get_code(&self, address: &Address, block: u64) -> Result<Vec<u8>> {
-        /*
-        let block = Some(BlockId::from(block));
-        let code = self
-            .provider
-            .get_code(*address, block)
-            .await
-            .map_err(|e| RpcError::new("get_code", e))?;
+        let address = serde_json::to_value(address)?;
+        let block = serde_json::to_value(BlockId::from(block))?;
 
-        Ok(code.to_vec())
-        */
-        todo!()
+        Ok(self
+            .request("eth_getCode", [address, block])
+            .await
+            .map_err(|e| RpcError::new("get_code", e))?)
     }
 
     async fn send_raw_transaction(&self, bytes: &[u8]) -> Result<H256> {
-        /*
         let bytes = Bytes::from(bytes.to_owned());
-        let tx = self
-            .provider
-            .send_raw_transaction(bytes)
-            .await
-            .map_err(|e| RpcError::new("send_raw_transaction", e))?;
 
-        Ok(tx.tx_hash())
-        */
-        todo!()
+        Ok(self
```

Comment:
I think you need to call `serde_json::to_value` before this

Reply:
manual calls to `to_value` are needed only if there is more than one value, to have uniform types in array